x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,352 @@
+                          <EuiButton
+                            fill
+                            isDisabled={runningDataRecognizer}
+                            isLoading={runningDataRecognizer}
+                            color={'primary'}
+                            onClick={runDataRecongizer}
+                            size="s"
+                          >
+                            <FormattedMessage
+                              id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.runDataRecognizerButtonLabel"
+                              defaultMessage="Run data recognizer"
+                            />
+                          </EuiButton>
+                        </span>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </EuiFlexItem>
+                ) : null}
+                {matchingDataViews.length ? (
+                  <EuiFlexItem grow={false}>
+                    <DataViewsTable matchingDataViews={matchingDataViews} moduleId={module.id} />
+                  </EuiFlexItem>
+                ) : null}
+              </EuiFlexGroup>
+            ) : (
+              <EuiEmptyPrompt
+                css={{ margin: '10px' }}
+                color="primary"
+                iconType={(isLogoObject(module.logo) ? module.logo?.icon : module.logo) as string}
+                title={
+                  <h2>
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.unableToUseModuleTitle"
+                      defaultMessage="You cannot create these jobs here"
+                    />
+                  </h2>
+                }
+                body={
+                  <p>
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.unableToUseModuleHelpMessage"
+                      defaultMessage="These supplied configurations can be used in {appName}."
+                      values={{ appName: module.type }}
+                    />
+                  </p>
+                }
+                footer={
+                  <>
+                    <EuiTitle size="xxs">
+                      <h3>
+                        <FormattedMessage
+                          id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.useInanotherAppTitle"
+                          defaultMessage="Want to learn more?"
+                        />
+                      </h3>
+                    </EuiTitle>
+                    <EuiLink
+                      href={module.type === 'Logs' ? logsConfigsUrl : metricsConfigsUrl}
+                      target="_blank"
+                    >
+                      <FormattedMessage
+                        id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.useInanotherAppLink"
+                        defaultMessage="Read the docs for more information on how to use this module in other apps"
```

Comment:
```suggestion
                        defaultMessage="Refer to the docs for more information on how to use this module in other apps"
```

Reply:
Updated in 9838277f8bb24511c72c3af4da087cbdb52624e7